### PR TITLE
PIMS-1514 Adding Parcel.Municipality

### PR DESCRIPTION
**Pull request by Fosol**
Fixing Parcel.ZoningPotential
New DB Migration - Requires DB Refresh!

### CODE REVIEW

File: backend/dal/Configuration/ParcelConfiguration.cs [hunk 1]
```diff
@@ -21,8 +21,9 @@ public override void Configure(EntityTypeBuilder<Parcel> builder)
             builder.Property(m => m.Description).HasMaxLength(2000);
             builder.Property(m => m.Latitude).IsRequired();
             builder.Property(m => m.Longitude).IsRequired();
-            builder.Property(m => m.Zoning).HasMaxLength(500);
-            builder.Property(m => m.ZoningPotential).HasMaxLength(500);
+            builder.Property(m => m.Municipality).HasMaxLength(250);
+            builder.Property(m => m.Zoning).HasMaxLength(250);
```
Comment (Fosol) [inline backend/dal/Configuration/ParcelConfiguration.cs:25]: 500 is too large for this field.  I've reduced to 250.

File: backend/dal/Configuration/ParcelConfiguration.cs [hunk 2]
```diff
@@ -21,8 +21,9 @@ public override void Configure(EntityTypeBuilder<Parcel> builder)
...
+            builder.Property(m => m.ZoningPotential).HasMaxLength(250);
```
Comment (Fosol) [inline backend/dal/Configuration/ParcelConfiguration.cs:26]: 500 is too large for this field.  I've reduced to 250.

File: backend/entities/Parcel.cs [hunk 3]
```diff
@@ -105,7 +110,7 @@ public class Parcel : BaseEntity
         /// <summary>
         /// get/set - Potential future Parcel zoning information
         /// </summary>
-        public bool ZoningPotential { get; set; }
+        public string ZoningPotential { get; set; }
```
Comment (Fosol) [inline backend/entities/Parcel.cs:113]: Fix bug as this should be a string, not boolean.

File: backend/tests/unit/api/Controllers/Admin/ParcelControllerTest.cs [hunk 4]
```diff
@@ -318,6 +326,45 @@ public void GetParcelByPid_String_NoContent()
...
+            service.Setup(m => m.Parcel.Get(It.IsAny<int>())).Returns(parcel);
+
+            // Act
+            var result = controller.GetParcel(parcel.Id);
+
+            // Assert
+            var actionResult = Assert.IsType<JsonResult>(result);
+            Assert.Null(actionResult.StatusCode);
+            var actualResult = Assert.IsType<Model.ParcelModel>(actionResult.Value);
+            Assert.Equal(parcel.Id, actualResult.Id);
```
Comment (Fosol) [inline backend/tests/unit/api/Controllers/Admin/ParcelControllerTest.cs:349]: Validating property output of model.

File: backend/tests/unit/api/Controllers/ParcelControllerTest.cs [hunk 5]
```diff
@@ -185,6 +186,54 @@ public void GetParcel_Success()
...
+            int expectedParcelId = 1;
+
+            // Act
+            var result = controller.GetParcel(expectedParcelId);
+
+            // Assert
+            var actionResult = Assert.IsType<JsonResult>(result);
+            Assert.Null(actionResult.StatusCode);
+            var actualResult = Assert.IsType<Model.ParcelModel>(actionResult.Value);
+            Assert.Equal(parcel.Id, actualResult.Id);
```
Comment (Fosol) [inline backend/tests/unit/api/Controllers/ParcelControllerTest.cs:218]: Validating property output of model.

File: backend/tests/unit/dal/Migrations/MigrationTest.cs [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+        #region Tests
+        /// <summary>
+        /// Ensure the first migration is called 'Initial'.
+        /// </summary>
+        [Fact]
+        public void InitialMigration()
+        {
+            // Arrange
+            var initial = new Initial();
```
Comment (Fosol) [inline backend/tests/unit/dal/Migrations/MigrationTest.cs:24]: Making sure this migration exists for the future.

File: backend/entities/Parcel.cs (hunk 3 above)
Comment (devinleighsmith) [reply backend/entities/Parcel.cs:113]: Thanks for fixing this.

**Pull request merged**